### ISSUE TITLE
Ability to disable kindaVim on certain websites

### ISSUE THREAD

**Issue by dkarter** [godbout/kindaVim.docs#180]
Hey!

I use kindaVim in Brave Browser, and every time I open a website that has Vim keybindings in an editor, kindaVim gets in the way of the editor (e.g. [Code Sandbox](https://codesandbox.io/)).

This makes it so I have to press escape twice every time and really makes editing difficult on some editors.

Would it be possible to disable kindaVim based on a browser title matching text? in the case of Code Sandbox it would be `CodeSandbox`

![CleanShot 2023-05-26 at 01 36 41@2x](https://github.com/godbout/kindaVim.docs/assets/551858/9efb6e58-0156-4359-9b60-1cbd0b0b4822)

Thanks for your consideration!

**Comment by godbout** [comment]
yeah, that's a very fair request. not possible currently in kV, and it's gonna be a massive undertake. very custom. i curate every app so that users don't have to do their own research. but here we're talking about **TABS**. that's a little insane, although from a user's POV that's a total fair request. so it would definitely make sense to implement this.

now, thinking out loud, what's the best UX?
1. obviously i can't enforce the Off Family depending on a tab title, so users will have to do that themselves ☹️
2. detection of browsers can be curated by me
3. add an easy way to send a page title to kV's Settings (would require a new setting of course)? like what we do currently with apps?

the whole thing definitely feels like a fragile area. but again, the idea makes total sense.

**Comment by godbout** [comment]
btw personally i don't really have this need because i use Safari for my daily tasks, and Brave for coding. so Off Family for Brave, PGR for Safari. workflow is good for me.
